news-bundle/src/Resources/contao/models/NewsModel.php

```diff
@@ -244,8 +244,10 @@ public static function findPublishedByPids($arrPids, $blnFeatured=null, $intLimi
 			$arrColumns[] = "$t.featured=''";
 		}
 
+		$blnShowUnpublished = System::getContainer()->get('contao.security.token_checker')->isPreviewMode();
+
 		// Never return unpublished elements in the back end, so they don't end up in the RSS feed
-		if (!BE_USER_LOGGED_IN || TL_MODE == 'BE')
+		if (!$blnShowUnpublished || TL_MODE == 'BE')
```

Comment:
```suggestion
		if (!static::isPreviewMode() || TL_MODE == 'BE')
```

Then you can remove line 247 again.

Reply:
Changed in 5fe28dd1daa178ebd9601d8822231f79302d2d37. This adds support for the `ignoreFePreview` option, but I think that's ok.